server/reports/expiring-items/2_3_0/argument_schemas/arguments_ui.json

```diff
@@ -57,7 +57,7 @@
     {
       "type": "Control",
       "scope": "#/properties/sort",
-      "label": "Sort by:",
+      "label": "T#report.sort-by",
```

Comment:
I realise missed the 'Sort by',  and 'Sort direction' keys and their translations - so have added these in also